tests/test_argument_check.py

```diff
@@ -31,7 +31,7 @@ def create_settings(test_type, vcs_type):
         settings.TeamcityServer.user_id = "TeamCityUser"
         settings.TeamcityServer.passwd = "TeamCityPassword"
 
-    if vcs_type in ("git", "gerrit", "github",):
+    if vcs_type in ["git", "gerrit", "github"]:
```

Comment:
Why you changed tuple to list?

Reply:
Because I think it is a little bit cleaner